app/controllers/health/certs_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Health
+  class CertsController < ApplicationController
+    newrelic_ignore_apdex
+
+    def index
+      deadline = params[:deadline].present? ? Time.zone.parse(params[:deadline]) : 30.days.from_now
+
+      result = health_checker.check_certs(deadline: deadline)
+
+      render json: result.as_json,
+             status: result.healthy? ? :ok : :internal_server_error
```

Comment:
Could we pick something like `:service_unavailable` (503) instead to avoid potential collisions with this endpoint crashing?

Reply:
Sure, done in 726c36e2a333ed68dd30716c464d242814345cdb